lib/public/views/About/About.js

```diff
@@ -11,19 +11,88 @@
  * or submit itself to any jurisdiction.
  */
 
-import { Observable } from '/js/src/index.js';
+import { getRemoteData } from '../../utilities/fetch/getRemoteData.js';
+import { Observable, RemoteData } from '/js/src/index.js';
 
 /**
- * Model representing handlers for homePage.js
+ * Model representing handlers for aboutPage.js
  */
-export default class Overview extends Observable {
+export class AboutModel extends Observable {
     /**
-     * The constructor of the Overview model object
-     * @param {Object} model Pass the model to access the defined functions
-     * @returns {Object} Constructs the Overview model
+     * The constructor for the About model object
+     * @param {Object} model Pass the parent model to access the defined functions
+     * @returns {Object} Constructs the About model
      */
     constructor(model) {
         super();
         this.model = model;
+        this._appInfo = RemoteData.NotAsked();
+        this._dbInfo = RemoteData.NotAsked();
+    }
+
+    /**
+     * Fetches information about the all services related to bookkeeping by querying the status api
+     * @returns {void}
+     */
+    async fetchAllServiceInfo() {
+        this._dbInfo = await this.fetchServiceInfo('/api/status/database');
+        this._appInfo = await this.fetchServiceInfo('/api/status/gui');
+        this.notify();
+    }
+
+    /**
+     * Fetches information about the service determined by the api path
+     * @param {string} apiPath the path to the endpoint for this service's status
+     * @returns {Object} an object which contains the version (if the request was successful)
+     *                   and the status of the service
+     */
+    async fetchServiceInfo(apiPath) {
+        let service = RemoteData.loading();
```

Comment:
This method should also receive by reference the serviceInfo object so that the changes from `loading` to `success` or `error` are visible in the UI.
Currently you create a variable within this scope only and also, you do not notify the model on the variable change.